tests/encoding_test.c

```diff
@@ -28,34 +28,38 @@ static int run_hex_encoding_test_case(struct aws_allocator *alloc, const char *t
     ASSERT_INT_EQUALS(expected_size, output_size, "Output size on string should be %d", expected_size);
 
     struct aws_byte_buf to_encode = aws_byte_buf_from_c_str(test_str, test_str_size - 1);
-    struct aws_byte_buf output;
-    ASSERT_SUCCESS(aws_byte_buf_init(alloc, &output, output_size + 2));
-    memset(output.buffer, 0xdd, output_size + 2);
 
-    output.capacity = output_size;
-    output.buffer += 1;
+    struct aws_byte_buf allocation;
```

Comment:
These changes look great. I would make the same changes to run_base64_encoding_test_case and it will be good to go.

Reply:
ok